### ISSUE TITLE
Release schedule for Scalatra 3.0

### ISSUE THREAD

**Issue by mvestola** [scalatra/scalatra#1559]
I could not find anywhere any info about the schedule of Scalatra 3.0 development so maybe it could be posted somewhere or commented in this issue. Are there any plans to release stable 3.0 version soon? How far is `3.0.0-M5-jakarta` release from the stable version? How many milestone releases will there still be until it is considered stable?

Plans can of course change but it would be nice to have some kind of rough schedule for stable 3.0 release so that people can make plans when would it be good to update Scalatra version in their applications.

**Comment by SvenssonWeb** [comment]
According to the developers on Gitter, Scalatra 3 is waiting on the release of Play Twirl 1.6.0 for Scala 3 support:
https://app.gitter.im/#/room/#scalatra_scalatra:gitter.im/$9xLN44ihJDB8lEktcWlT5aRbLAwW3l4bD0zkx_lBGkE

You can find a ticket on Twirl for 1.6.0 release here: https://github.com/playframework/twirl/issues/628

**Comment by takezoe** [comment]
Twirl 1.6.0 has been finally released. We will release Scalatra 3.0.0 soon.

**Comment by takezoe** [comment]
https://github.com/scalatra/scalatra/releases/tag/v3.0.0

**Comment by yamachu** [comment]
Thanks for releasing scalatra v3.0.0 by this PR (https://github.com/scalatra/scalatra/pull/1576)!

I was trying to try v3.0.0 on our products, but it looks like it has not been published on maven yet.
What are your plans to publish core packages and related packages (e.g. scalatra-json)?
Thanks!

**Comment by takezoe** [comment]
@yamachu As mentioned in [README](https://github.com/scalatra/scalatra/blob/main/README.markdown#latest-version), artifact names are now like below:

- scalatra-core-javax
- scalatra-core-javaee

**Comment by mvestola** [comment]
To switch from milestone release to latest stable release, I changed packages like this in my sbt dependencies:

Previously:
```
private val scalatraVersion = "3.0.0-M5-jakarta"
lazy val scalatra = "org.scalatra" %% "scalatra" % scalatraVersion
lazy val scalatraJson = "org.scalatra" %% "scalatra-json" % scalatraVersion
lazy val scalatraSwagger = "org.scalatra" %% "scalatra-swagger" % scalatraVersion
lazy val scalaTest = "org.scalatra" %% "scalatra-scalatest" % scalatraVersion % "test"
```

Now:
```
private val scalatraVersion = "3.0.0"
lazy val scalatra = "org.scalatra" %% "scalatra-jakarta" % scalatraVersion
lazy val scalatraJson = "org.scalatra" %% "scalatra-json-jakarta" % scalatraVersion
lazy val scalatraSwagger = "org.scalatra" %% "scalatra-swagger-jakarta" % scalatraVersion
lazy val scalaTest = "org.scalatra" %% "scalatra-scalatest-jakarta" % scalatraVersion % "test"
```

All seemed to work :+1:

**Comment by yamachu** [comment]
@takezoe Thank you for replying.

I was adding -javax only to the `scalatra`, but I solved the problem by adding the `-javax` suffix to the other dependencies as @mvestola  presented. (Thanks, @mvestola !)

**Comment by frou** [comment]
Version 3.0 didn't get a news item on the website 😢 
<img width="264" alt="Screenshot 2023-12-30 at 10 54 02" src="https://github.com/scalatra/scalatra/assets/172663/d927c8d2-000f-4c6c-85f5-f32682e92e26">

**Comment by takezoe** [comment]
Ah, yes. We didn't write an official blog post this time.
